Documentation/Tutorials/edge-send-event-tutorial.md

```diff
@@ -0,0 +1,668 @@
+
+Select **New Datastream** in the top right.
+
+<img src="../Assets/edge-send-event-tutorial/datastreams-main-view.png" alt="Create new datastream" width="1100"/>  
+
+Give the datastream an identifying name and description (**1**), then pick the schema created in the previous section using the dropdown menu (**2**). Then select **Save** (**3**).
+
+<img src="../Assets/edge-send-event-tutorial/datastreams-new-datastream.png" alt="Set datastream values" width="1100"/>  
+
+With the datastream set up, data can be directed to its destination by adding services. In this case, Adobe Analytics:
+1. Select **Add Service** (**1**)
+
+<img src="../Assets/edge-send-event-tutorial/datastreams-add-service.png" alt="Set datastream values" width="1100"/>  
+
+2. From the **Service (required)** dropdown (**1**), select **Adobe Analytics**.
+3. Select **Add Report Suite** (**2**), and enter the report suite ID you want the data from this tutorial to land.
+4. Select **Save**.
+
+<img src="../Assets/edge-send-event-tutorial/datastreams-add-analytics.png" alt="Set datastream values" width="1100"/>  
+
+### 3. Create a property
+
+Next, create a property for mobile. A property is basically the configuration settings for AEP extensions, allowing you to control their functionality. 
+
+Select **Tags** (**1**) under **DATA COLLECTION** in the left-side navigation panel.
+
+<img src="../Assets/edge-send-event-tutorial/data-collection-tags-navigation.png" alt="Navigating to tags" width="1100"/>  
+
+Select **New Property** (**1**) to create a new property.
+
+<img src="../Assets/edge-send-event-tutorial/tags-main-view.png" alt="Navigating to tags" width="1100"/>  
+
+Enter an identifying name for the new property in the **Name** textfield (**1**), select **Mobile** (**2**) under **Platform**, then select **Save** (**3**).
+
+<img src="../Assets/edge-send-event-tutorial/tags-create-property.png" alt="Navigating to tags" width="1100"/>  
+
+Find and select the mobile property for this tutorial (**2**), optionally using the search box to help quickly narrow down the search (**1**).
+
+<img src="../Assets/edge-send-event-tutorial/property-search.png" alt="Finding desired mobile property" width="1100"/>  
+
+Select **Extensions** (**2**) in the left-side navigation panel, under **AUTHORING**. Notice there are some extensions are that installed by default (**1**).
+
+<img src="../Assets/edge-send-event-tutorial/mobile-property-extensions.png" alt="Finding desired mobile property" width="1100"/>  
+
+Select **Catalog** (**1**) and (optionally) use the search box (**2**) to find the required extensions; select the **Install** button in an extension card to install the extension. 
+
+<img src="../Assets/edge-send-event-tutorial/mobile-property-catalog.png" alt="Catalog search example" width="1100"/>  
+
+We will be installing the following AEP extension configurations:
```

Comment:
"We will be installing the following AEP extension configurations" -> "Install the AEP extensions with the following configurations"